pkg/actors/actors.go

```diff
@@ -10,6 +10,7 @@ import (
 	"encoding/json"
 	"errors"
 	"fmt"
+	"math/rand"
```

Comment:
PS: I have updated the bug with my findings on where the fix was. The idea is to be able to distinguish old reminder values using a new field ID. This would be a breaking change since persisted reminders will not have the ID field.